src/shrutiC/FindSecondMaxNum.java

```diff
@@ -0,0 +1,29 @@
+package shrutiC;
+
+public class FindSecondMaxNum {
+
+	void getSecoundMaxNumInArray(int arrayNum[]) {
+		int max = arrayNum[0];
+		int secondMaxNum = 0;
+		System.out.print("Input array : {" + max);
+		for (int i = 1; i < arrayNum.length; i++) {
+			System.out.print(" " + arrayNum[i]);
+			if (arrayNum[i] > max) {
+				secondMaxNum = max;
+				max = arrayNum[i];
+			}
```

Comment:
you need to put else if() condition as well
else if (input[index] > secondMax && input[index] != max)